Libraries/Network/RCTHTTPRequestHandler.mm

```diff
@@ -60,10 +60,23 @@ - (NSURLSessionDataTask *)sendRequest:(NSURLRequest *)request
 {
   // Lazy setup
   if (!_session && [self isValid]) {
+    // You can override default NSURLSession instance property allowsCellularAccess (default value YES)
+    //  by providing the following key to your RN project (edit ios/project/Info.plist file in Xcode):
+    // <key>ReactNetworkForceWifiOnly</key>    <string>YES</string>
+    // This will set allowsCellularAccess to NO and force Wifi only for all network calls on iOS
+    // If you do not want to override default behavior, do nothing or set key with value "NO"
+    NSDictionary *infoDictionary = [[NSBundle mainBundle] infoDictionary];
+    NSString *useWifiOnly = [infoDictionary objectForKey:@"ReactNetworkForceWifiOnly"];
```

Comment:
@cpojer Can we change this to Bool? we just have two values YES or NO, don't need to do string comparison.

Reply:
I don't have strong feelings on it. @ericlewis @bondparkerbond what do you think? If you think we should change this, please send a follow-up PR.

Reply:
@ericlewis @bondparkerbond Here we go, please see https://github.com/facebook/react-native/pull/24829.